packages/paste-core/components/button/src/styles.ts

```diff
@@ -73,4 +73,13 @@ export const SizeStyles: {[key in ButtonSizes]: BoxStyleProps} = {
     borderWidth: 'borderWidth0',
     fontSize: 'inherit',
   },
+  rounded: {
+    borderRadius: 'borderRadiusPill',
+    paddingTop: 'space30',
```

Comment:
Question: on Figma, it looks like the button is 28px in height; is there a way to get to that height by adjusting this padding? Or is there no combination that makes it possible token-wise?

Reply:
Suggestion: I think it would match the design better to use `space20` for the top and bottom and `space30` for left and right.

Reply:
`paddingX: space40; paddingY: space30` (current):
<img width="177" alt="Screen Shot 2022-06-27 at 9 48 30 AM" src="https://user-images.githubusercontent.com/75342690/175959360-6b68fe56-df3d-4ef3-a5ed-70be75b27d9b.png">

`paddingX: space30; paddingY: space20`:
<img width="159" alt="Screen Shot 2022-06-27 at 9 57 06 AM" src="https://user-images.githubusercontent.com/75342690/175959395-20f43a00-e452-4473-ae2f-e08f8d4daa6f.png">

Figma:
<img width="202" alt="Screen Shot 2022-06-27 at 9 59 41 AM" src="https://user-images.githubusercontent.com/75342690/175959480-c5a46b9b-b03c-436a-87c5-e1b81a15caf1.png">

@libertycarlin let me know which one to go with! or we can hop on another zoom today to play around with the padding